### ISSUE TITLE
CoreDescriptors and CoreSchedules do not use a transparent hasher for keys

### ISSUE THREAD

**Issue by seadanda** [paritytech/polkadot-sdk#6563]
Both use `Twox256` instead of e.g. `Twox64Concat`, obfuscating the value of the keys.

from a conversation [here](https://github.com/polkadot-api/papi-console/issues/2#issuecomment-2489038557)

Suggest using either `Twox64Concat` or `Blake2_128Concat` for the keys

**Comment by seadanda** [comment]
I think about it the other way round: why use one that isn't reconstructable without reason?
In this case I get that we don't need to iterate them, but I think UIs and anybody reading the state would find it useful to have readable keys.

Happy either way, especially since this is already live and would be a breaking change to change it. I could easily be convinced it's not worth the change, but it's definitely a minor inconvenience.

**Comment by eskimor** [comment]
Good catch @seadanda ! Will be fixed as part of #1312 . ( I am changing the storage there anyway)
